### PR TITLE
Adds integration tests for individual commands.

### PR DESCRIPTION
**Pull request by flosincapite**

### CODE REVIEW

**Comment by flosincapite** [comment]
This is a WIP, but I want people to check it out in case this testing strategy looks like skulduggery.

**Comment by flosincapite** [comment]
Does not cover all commands yet, but I'd like to submit it so we can increase the coverage.

File: tests/integration/cronenbroguelike/commands_test.py [hunk 1]
```diff
@@ -0,0 +1,135 @@
+import io
+import unittest
```
Comment (xmunoz) [inline tests/integration/cronenbroguelike/commands_test.py]: This isn't used, please remove.

File: tests/integration/cronenbroguelike/commands_test.py [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+from unittest.mock import call
+from unittest.mock import Mock
```
Comment (xmunoz) [inline tests/integration/cronenbroguelike/commands_test.py]: This isn't used, please remove.

File: tests/integration/cronenbroguelike/commands_test.py [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+from unittest.mock import patch
+
+import adventurelib
+
+from cronenbroguelike import ability
+from cronenbroguelike import commands
```
Comment (xmunoz) [inline tests/integration/cronenbroguelike/commands_test.py]: This isn't used, please remove.

File: tests/integration/cronenbroguelike/commands_test.py [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+                call("stamina   : 10", add_newline=False),
+                call("------", add_newline=False),
+                call("- abilities -", add_newline=False),
+            ]
+        )
+
+    def test_different_stats(self, mock_say, mock_stdin):
+        G.player = actor.create_actor(20, 20, 20, 20, "the flippin devil")
+        G.player.health.heal_or_harm(-9)
+        G.player.add_ability(ability.meditation)
```
Comment (xmunoz) [inline tests/integration/cronenbroguelike/commands_test.py]: Should this be:
```
         G.player.add_ability(ability.meditation())
```
?
If so, how is this passing right now?

**Review by xmunoz — CHANGES_REQUESTED** [review]

File: tests/integration/cronenbroguelike/commands_test.py (hunk 3 above)
Comment (flosincapite) [reply tests/integration/cronenbroguelike/commands_test.py]: It is used implicitly. Importing commands registers all the `@when`-decorated functions; without this line, the whole house of cards would come down.

File: tests/integration/cronenbroguelike/commands_test.py (hunk 4 above)
Comment (flosincapite) [reply tests/integration/cronenbroguelike/commands_test.py]: It worked for a very bad reason. `abilities.py` will be whipped into shape as part of the serialization/deserialization refactor.

Comment (flosincapite) [reply tests/integration/cronenbroguelike/commands_test.py]: I have made a comment on an open issue that will address this problem.

File: tests/integration/cronenbroguelike/commands_test.py (hunk 3 above)
Comment (xmunoz) [reply tests/integration/cronenbroguelike/commands_test.py]: Can you leave a comment to that effect in the code? This is unintuitive/unexpected.

**Review by xmunoz — APPROVED** [review]
Just a small nit. LGTM

File: tests/integration/cronenbroguelike/commands_test.py (hunk 3 above)
Comment (flosincapite) [reply tests/integration/cronenbroguelike/commands_test.py]: Yes, and I added a pragma so your linter doesn't complain.

**Pull request merged**